airgun/views/discoveryrule.py

```diff
@@ -36,6 +36,19 @@ def is_displayed(self):
             self.title, exception=False) is not None
 
 
+class SearchInput(TextInput):
+    """Discovery Search input field, This field has no value attribute, and The only way to set or
+    read it values correctly is using javascript.
```

Comment:
we can't even read value?

Reply:
Yes the standard read, return the value from the list and not the current visible one.